docs/_posts/luca-martial/2022-02-22-clinical_deidentification_fr.md

```diff
@@ -0,0 +1,230 @@
+L'examen physique a montré un bon état général, avec un abdomen et des organes génitaux normaux ; le toucher rectal était compatible avec un adénome de la prostate de grade I/IV.
+L'analyse d'urine a montré 4 globules rouges/champ et 0-5 leucocytes/champ ; le reste du sédiment était normal.
+Hémogramme normal ; la biochimie a montré une glycémie de 169 mg/dl et des triglycérides de 456 mg/dl ; les fonctions hépatiques et rénales étaient normales.
+PSA de 1,16 ng/ml.
+ADDRESSÉ À : [**************] - [****************************] Service D'Endocrinologie et de Nutrition - [******************], [***] [******]
+COURRIEL : [****************]
+
+
+Masked with fixed length chars
+------------------------------
+COMPTE-RENDU D'HOSPITALISATION
+PRENOM : ****
+NOM : ****
+NUMÉRO DE SÉCURITÉ SOCIALE : ****
+ADRESSE : ****
+VILLE : ****
+CODE POSTAL : ****
+DATE DE NAISSANCE : ****
+Âge : **** 
+Sexe : ****
+COURRIEL : ****
+DATE D'ADMISSION : ****
+MÉDÉCIN : ****
+RAPPORT CLINIQUE : **** ans, retraité, sans allergie médicamenteuse connue, qui présente comme antécédents : ancien accident du travail avec fractures vertébrales et des côtes ; opéré de la maladie de Dupuytren à la main droite et d'un pontage ilio-fémoral gauche ; diabète de type II, hypercholestérolémie et hyperuricémie ; alcoolisme actif, fume 20 cigarettes / jour.
+**** nous a été adressé car **** présentait une hématurie macroscopique postmictionnelle à une occasion et une microhématurie persistante par la suite, avec une miction normale.
+L'examen physique a montré un bon état général, avec un abdomen et des organes génitaux normaux ; le toucher rectal était compatible avec un adénome de la prostate de grade I/IV.
+L'analyse d'urine a montré 4 globules rouges/champ et 0-5 leucocytes/champ ; le reste du sédiment était normal.
+Hémogramme normal ; la biochimie a montré une glycémie de 169 mg/dl et des triglycérides de 456 mg/dl ; les fonctions hépatiques et rénales étaient normales.
+PSA de 1,16 ng/ml.
+ADDRESSÉ À : **** - **** Service D'Endocrinologie et de Nutrition - ****, **** ****
+COURRIEL : ****
+
+
+Obfuscated
+------------------------------
+COMPTE-RENDU D'HOSPITALISATION
+PRENOM : M. Martine Vincent-Perrin
+NOM : M. Christiane-Susan Launay
+NUMÉRO DE SÉCURITÉ SOCIALE : 179057055072335
+ADRESSE : boulevard Bertrand
+VILLE : Gilbert
+CODE POSTAL : 59474
+DATE DE NAISSANCE : 16/04/1946
+Âge : 39 ans 
+Sexe : F
+COURRIEL : npoirier@tele2.fr
+DATE D'ADMISSION : 08/01/2017
+MÉDÉCIN : Dre. Martine Da Costa
+RAPPORT CLINIQUE : de l'age de 7 ans ans, retraité, sans allergie médicamenteuse connue, qui présente comme antécédents : ancien accident du travail avec fractures vertébrales et des côtes ; opéré de la maladie de Dupuytren à la main droite et d'un pontage ilio-fémoral gauche ; diabète de type II, hypercholestérolémie et hyperuricémie ; alcoolisme actif, fume 20 cigarettes / jour.
```

Comment:
I think for AGE it's better to obfuscated with just a number, don't include strings if your <AGE> detects only the number. Because you are getting <AGE> ans, and then you obfuscate <AGE> as 7 ans or de l'age de 7 ans, and you get `de l'age de 7 ans ans`

Reply:
great call @josejuanmartinez thank you! i will reupload